src/librustdoc/clean/cfg.rs

```diff
@@ -515,6 +515,7 @@ impl<'a> fmt::Display for Display<'a> {
                         "aarch64" => "AArch64",
                         "arm" => "ARM",
                         "asmjs" => "JavaScript",
+                        "loongarch64" => "LoongArch64",
```

Comment:
Would "LoongArch LA64" be better?

Reply:
Loongarch64 is used here, to distinguish it when considering later adaptation to 32-bit architecture.
Maybe we can also ask xen0n’s opinion.(@xen0n)

Reply:
I'll try to find some time to rewrite the doc, unfortunately the past few weeks were full of `$DAY_JOB` paperwork. glibc release is near, this certainly doesn't help either. (taking a sip of coffee ☕)

As for "LoongArch64" or "LoongArch LA64", while I know "LoongArch64" doesn't appear in official documents it's certainly shorter. At this point I don't have a strong opinion but I'd like to remind @zhaixiaojuan that the "LA64" also serves to distinguish 64-bit from 32-bit, plus it's explicitly mentioned in the ISA manual as a LoongArch subset (well, complete set with regard to base ISA) apart from LA32 and LA32 Primary.